### PR TITLE
Fix Card global css box sizing

### LINKED ISSUE #193: Card css is targeting global wildcard pseudo elements
Card css shouldn't target wildcard all pseudo elements 

```css 
.uitkCard *, *:before, *:after {
    box-sizing: inherit;
}
```

### PR DESCRIPTION
**Pull request by EleanorHaproff**
fixes #193 

`<Card/>` had some css which set things globally which has now been removed. (No component-level css files should do this)

In addition, `box-sizing: border-box` is now set globally properly in `/theme/global.css`, and removed from components.

### CODE REVIEW

**Review by joshwooding — APPROVED** [review]

**Review by joshwooding — CHANGES_REQUESTED** [review]
Can we add a changeset please :)

File: packages/ag-grid-theme/css/_uitk-ag-theme-mixin.scss [hunk 1]
```diff
@@ -571,7 +571,6 @@
 
   [class^="ag-"]:focus {
     outline: none;
-    box-sizing: border-box;
```
Comment (joshwooding) [inline packages/ag-grid-theme/css/_uitk-ag-theme-mixin.scss:574]: @ararus Do these changes make sense to you?

File: .changeset/healthy-coats-rule.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+"docs": minor
```
Comment (joshwooding) [inline .changeset/healthy-coats-rule.md:2]: Can we remove the docs package from here please

File: packages/ag-grid-theme/css/_uitk-ag-theme-mixin.scss (hunk 1 above)
Comment (ararus) [reply packages/ag-grid-theme/css/_uitk-ag-theme-mixin.scss:574]: It should be fine to move it to global, looks ok

**Pull request merged**